src/domain/safe/entities/schemas/erc20-transfer.schema.ts

```diff
@@ -1,5 +1,21 @@
 import { Schema } from 'ajv';
+import { z } from 'zod';
+import { AddressSchema } from '@/validation/entities/schemas/address.schema';
+import { HexSchema } from '@/validation/entities/schemas/hex.schema';
 
+export const Erc20TransferSchema = z.object({
+  type: z.literal('ERC20_TRANSFER'),
+  executionDate: z.coerce.date(),
+  blockNumber: z.number(),
+  transactionHash: HexSchema,
+  to: AddressSchema,
+  from: AddressSchema,
+  value: z.string(),
+  tokenAddress: AddressSchema.nullish().default(null),
```

Comment:
This shouldn't be `null`. See https://github.com/safe-global/safe-transaction-service/blob/bb5c5c2f663fdd047cf4e74a9df0fb7da24403f7/safe_transaction_service/history/serializers.py#L845-L853

Reply:
Good catch. Changed in ee19ddaff145d1a91fca25d53853eb0c1d8b850c.